parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -522,6 +522,7 @@ installCNI() {
     os_version="${UBUNTU_RELEASE}"
     if isMarinerOrAzureLinux "${OS}" && [ "${IS_KATA}" = "true" ]; then
         os=${MARINER_KATA_OS_NAME}
+        DNF = 1
```

Comment:
open to better ways to tdo this.

Reply:
We have parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh and cse_install_mariner.sh for both OS. You can move your function to the files correspondingly

Reply:
btw, I think you will also need to update function `downloadCNI` because install-dependencies.sh is calling this function. This file is called during VHD build time.
cse_install.sh will be executed during provisioning time.